spec/v2.spec.ts

```diff
@@ -488,24 +750,6 @@ describe('v2', () => {
           expect(cloudEvent.ref).equal('foo/StringParam/baz');
         });
 
-        it.skip('should resolve default ref given TernaryExpression', () => {
```

Comment:
The test relies on a broken `.then(..)` method that no longer exists on the `CompareExpression` object. I could leave the test skipped instead of removing it and try to fix it in a different PR.

Reply:
Removing the test sounds good, I was just curious